cmd/bundle/run.go

```diff
@@ -23,8 +25,20 @@ var runCmd = &cobra.Command{
 	PreRunE: ConfigureBundleWithVariables,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		b := bundle.Get(cmd.Context())
+
+		if deployFlag {
+			err := deploy(cmd, b)
+			if err != nil {
+				return err
+			}
+		} else if computeID != "" {
+			// Running notebooks is not yet implemented, otherwise we could
+			// use --compute with a notebook
+			return fmt.Errorf("not supported: --compute specified without --deploy")
+		}
```

Comment:
Something else to consider: it can be confusing to users if they "run" without first doing "deploy." Maybe we should warn / error for that case if it's a "debug" environment?